testing/test_many.sh

```diff
@@ -32,10 +47,12 @@ echo DAQ stress test | tee -a $TEST_RESULTS
 cmd/run -b run_limit=$RUN_LIMIT settle_sec=0 dhcp_lease_time=120s
 cat inst/result.log
 results=$(fgrep [] inst/result.log | wc -l)
-timeouts=$(fgrep "dhcp:TimeoutError" inst/result.log | wc -l)
+timeouts=$(fgrep "ipaddr:TimeoutError" inst/result.log | wc -l)
+static_ips=$(fgrep done/-1 inst/cmdrun.log | wc -l)
 echo Found $results successful runs.
 # This is broken -- should have many more results available!
 echo Enough results: $(echo "print($results >= 6 / 10 * $RUN_LIMIT)" | python) | tee -a $TEST_RESULTS
-# $timeouts should strictly equal $NUM_NO_DHCP_DEVICES when dhcp step is fixed. 
-echo Enough DHCP timeouts: $(echo "print($timeouts >= $NUM_NO_DHCP_DEVICES)" | python) | tee -a $TEST_RESULTS
+# $timeouts should strictly equal $NUM_TIMEOUT_DEVICES when dhcp step is fixed. 
+echo Enough DHCP timeouts: $(echo "print($timeouts >= $NUM_TIMEOUT_DEVICES)" | python) | tee -a $TEST_RESULTS
```

Comment:
Nit: I'd recommend just "$((timeouts >= NUM_TIMEOUT_DEVICES))"  -- true the output is more cryptic (0 or 1), but the code itself is a lot easier to read...

Reply:
I couldn't get the the check before this (60% of run limit) to work in bash so I converted all of them to pass through python. If you just change this line, this line will output 1 where other lines will output True.

Reply:
$((results >= 6*RUN_LIMIT/10)) should work (just tried it in my shell) -- I think you were getting tripped up because it's doing integer arithmetic not floating point.  (6/10 == 0)  K - not critical, this python thing just seems ugly.   (Also not so concerned about the output being cryptic, as the code being more readable...)  Not a big deal...